app/models/participant_declaration.rb

```diff
@@ -1,30 +1,52 @@
 # frozen_string_literal: true
 
 class ParticipantDeclaration < ApplicationRecord
-  has_one :profile_declaration
+  has_many :profile_declarations
```

Comment:
Do we want to track fundable vs non-fundable as multiple profile_declarations? Is that easier than having a trackable fundability as an association within the profile declaration? I saw the profile declaration as being an extension to the participant_declaration so that we could add meta-data to it. Both mechanisms should work, but I think it merits further discussion.

Reply:
Do you mean `has_one :payable_profile_declaration`? `has_many :payable_profile_declarations`? 

I think that we just don't know how this stuff will be used exactly. I am happy to say that we will care about when declaration became payable, and so to put payable flag on profile declaration - but anything beyond this seems much too speculative for my taste.